source/documentBase.py

```diff
@@ -418,6 +506,30 @@ def script_lastColumn(self, gesture):
 	# Translators: the description for the last table column script on browseMode documents.
 	script_lastColumn.__doc__ = _("moves to the last table column")
 
+	def script_sayAllRow(self, gesture):
+		self._tableSayAll(_Movement.NEXT, _Axis.COLUMN)
+	script_sayAllRow.__doc__ = _(
+		# Translators: the description for the sayAll row command
+		"Reads row horizontally from current table cell up untill the last cell in the row."

```

Comment:
I have made suggestion for the script description in the user guide (see below). If you accept my suggestions, do not forget to align the script descriptions in this file.